app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_post_comment_count
+
+  private
+
+  def update_post_comment_count
+    post.comments_counter += 1
```

Comment:
- [ ] Line no. `10` is not necessary, As the `increment!` method is doing the same work.

Reply:
Thanks, changes have been made.